osbs/conf.py

```diff
@@ -491,3 +491,11 @@ def get_can_orchestrate(self):
     def get_info_url_format(self):
         return self._get_value("info_url_format", self.conf_section,
                                "info_url_format")
+
+    def get_artifacts_allowed_domains(self):
+        value = self._get_value("artifacts_allowed_domains", self.conf_section,
+                                "artifacts_allowed_domains")
+        if value:
+            return [x.strip() for x in ''.join(value.splitlines()).split(',')]
```

Comment:
Yes, that's much cleaner. Changed accordingly. Also, created a new commit for using the same approach for parsing `smtp_additional_addresses` and `smtp_error_addresses`.